job_definitions/manual.yml

```diff
@@ -15,8 +15,13 @@
           cron: "H/2 * * * *"
     builders:
       - shell: |
-          [ -x /tmp/dm-manual-venv/bin/pip ] || virtualenv /tmp/dm-manual-venv
-          . /tmp/dm-manual-venv/bin/activate
+          docker pull digitalmarketplace/manual || true
+          docker build --pull -t digitalmarketplace/manual .
+          docker push digitalmarketplace/manual
 
-          pip install -r requirements.txt
-          make clean html pages
+          docker run -it --rm \
+            -v $(pwd):/app \
+            digitalmarketplace/manual \
+            make clean html
+          
+          make pages
```

Comment:
`make pages` runs `make html` which runs `${VIRTUALENV_ROOT}/bin/sphinx-build -b html ARGS`

So it'll still need a venv here I think

Reply:
Good spot, I've added a commit to https://github.com/alphagov/digitalmarketplace-manual/pull/197 to remove the `make html` line from `buildpages`